lib/src/wizard.dart

```diff
@@ -112,7 +113,7 @@ class Wizard extends StatefulWidget {
   const Wizard({
     super.key,
     this.initialRoute,
-    required this.routes,
+    this.routes,
```

Comment:
should we have some kind of assert that either routes or controller is provided?